packages/css-library/src/stylesheets/lib/_normalize.scss

```diff
@@ -0,0 +1,13 @@
+/*
+  @use 'uswds-global';
+  - contains the following:
+    @forward "uswds-elements/lib/normalize";
+    @forward "uswds-core";
+    @forward "uswds-fonts";
+    @forward "uswds-elements";
+    @forward "uswds-helpers";
+
+  See: https://github.com/uswds/uswds/blob/98566ec5dc12807a2b0bc36699a033fe3a711d44/packages/uswds-global/_index.scss
+*/
+
+@use "uswds-elements/lib/normalize";
```

Comment:
I thought we could just start using the USWDS normalize import instead of housing it ourself. 

In Formation, it was imported from USWDS v1 which was using v3.0.3. USWDS v3 is using [v8.0.1](https://github.com/uswds/uswds/blob/98566ec5dc12807a2b0bc36699a033fe3a711d44/packages/uswds-elements/lib/_normalize.scss#L4)

Here is a diff of those two versions:

https://www.diffchecker.com/XH2TFwMU/

Reply:
I'm in favor of moving to V3's normalize.

Reply:
I agree with Andrew. Where impact is minimal, we should try and get onto V3 stuff as we go.